ksqldb-parser/src/main/java/io/confluent/ksql/parser/properties/with/CreateSourceAsProperties.java

```diff
@@ -180,6 +197,22 @@ public Map<String, String> getValueFormatProperties(final String valueFormat) {
       builder.put(ProtobufProperties.UNWRAP_PRIMITIVES, ProtobufProperties.UNWRAP);
     }
 
+    final String nullableRep = props.getString(
```

Comment:
Same comments as above, and also for CreateSourceProperties below.

Reply:
Done